app/components/Activity/ActivityModal/ActivityModal.js

```diff
@@ -42,11 +40,9 @@ export default class ActivityModal extends React.PureComponent {
     const SpecificModal = MODAL_COMPONENTS[item.type].component
 
     return (
-      <Modal onClose={hideActivityModal}>
-        <Box width={9 / 16} mx="auto">
-          <SpecificModal {...MODAL_COMPONENTS[item.type].props} />
-        </Box>
-      </Modal>
+      <Box width={9 / 16} mx="auto">
```

Comment:
Same goes here. positioning props would be better passed in from the caller.

Reply:
updated